cmd/versitygw/scoutfs.go

```diff
@@ -69,6 +71,12 @@ move interfaces as well as support for tiered filesystems.`,
 				EnvVars:     []string{"VGW_BUCKET_LINKS"},
 				Destination: &bucketlinks,
 			},
+			&cli.UintFlag{
+				Name:        "dir-perms",
+				Usage:       "default directory permissions for new directories",
+				EnvVars:     []string{"VGW_DIR_PERMS"},
+				Destination: &dirPerms,
+			},
```

Comment:
You can simply provide default value for `dirPerms` to avoid overriding it in scoutfs and posix initialization. 
e.g.
```go
&cli.UintFlag{
	Name:        "dir-perms",
	Usage:       "default directory permissions for new directories",
	EnvVars:     []string{"VGW_DIR_PERMS"},
	Destination: &dirPerms,
	Value:       0755,
},
```